code/modules/mob/inventory.dm

```diff
@@ -379,4 +379,6 @@ var/list/slot_equipment_priority = list( \
 //placeholder until tg inventory system
 /mob/proc/is_holding(obj/item/I)
 	return ((I == l_hand) || (I == r_hand))
-z
\ No newline at end of file
+
+/mob/proc/can_equip(obj/item/I, slot, disable_warning = FALSE, bypass_equip_delay_self = FALSE)
```

Comment:
what's this new parameter used for? i don't think it belongs here

Reply:
> what's this new parameter used for? i don't think it belongs here

Deleted